php/sqlcommenter-php/packages/sqlcommenter-laravel/src/Database/Connection.php

```diff
@@ -0,0 +1,117 @@
+<?php
+namespace Google\GoogleSqlCommenterLaravel\Database;
+
+use Illuminate\Database\Connection as BaseConnection;
+use OpenTelemetry\API\Trace\Propagation\TraceContextPropagator;
+use Google\GoogleSqlCommenterLaravel\Opentelemetry;
+use Google\GoogleSqlCommenterLaravel\Utils;
+
+
+class Connection extends BaseConnection
+{
+    /**
+     * Run a select statement and return a single result.
+     *
+     * @param  string  $query
+     * @param  array  $bindings
+     * @param  bool  $useReadPdo
+     * @return mixed
+     */
+    public function selectOne($query, $bindings = [], $useReadPdo = true)
+    {
+        $query .= $this->get_sqlcomments();
+        $records = parent::select($query, $bindings, $useReadPdo);
+
+        if (count($records) > 0)
+        {
+            return $records;
+        }
+        return null;
+    }
+
+    /**
+     * Run a select statement against the database.
+     *
+     * @param  string  $query
+     * @param  array  $bindings
+     * @param  bool  $useReadPdo
+     * @return array
+     */
+    public function select($query, $bindings = [], $useReadPdo = true)
+    {
+        $query .= $this->get_sqlcomments();
+        $records = parent::select($query, $bindings, $useReadPdo);
+        return $records;
+
+    }
+
+    /**
+     * Run an insert statement against the database.
+     *
+     * @param  string  $query
+     * @param  array  $bindings
+     * @return bool
+     */
+    public function insert($query, $bindings = [])
+    {
+        $query .= $this->get_sqlcomments();
+
+        $records = parent::insert($query, $bindings);
+
+        return $records;
+    }
+
+    /**
+     * Run an update statement against the database.
+     *
+     * @param  string  $query
+     * @param  array  $bindings
+     * @return int
+     */
+    public function update($query, $bindings = [])
+    {
+        $query .= $this->get_sqlcomments();
+
+        return $this->affectingStatement($query, $bindings);
+    }
+
+    /**
+     * Run a delete statement against the database.
+     *
+     * @param  string  $query
+     * @param  array  $bindings
+     * @return int
+     */
+    public function delete($query, $bindings = [])
+    {
+        $query .= $this->get_sqlcomments();
+
+        return $this->affectingStatement($query, $bindings);
+    }
+
+    public function get_sqlcomments()
+    {
+        $configurationKey = 'google_sqlcommenter.include.';
+        $comment = [];
+        if (config($configurationKey.'framework', true)){
+            $comment['framework'] = "laravel-" .app()->version();
+        }
+        if (config($configurationKey.'controller', true)){
+            $action = app('request')->route()->getAction();
```

Comment:
can `route()` be null ? or getAction() can return null value ?
Similar for other parameters used below.